### ISSUE TITLE
Fresh Install Error for Vendor Publish

### ISSUE THREAD

**Issue by g0ld3lux** [thedevdojo/chatter#17]
I got this error when i do php artisan vendor publish

```
  [League\Flysystem\FileNotFoundException]
  File not found at path: 016_07_29_171118_create_chatter_categories_table.php
```

**Comment by g0ld3lux** [comment]
I manually copy paste the file in the vendor to database to fix this...
but i hope this would be fix for laravel 5.3

**Comment by marktopper** [comment]
I can not recreate this error, might have been solved.
